ORM/ArrayList.php

```diff
@@ -1,4 +1,14 @@
 <?php
+
+namespace SilverStripe\ORM;
+
+use ViewableData;
+use ArrayData;
+use ArrayIterator;
+use Debug;
+use InvalidArgumentException;
+use LogicException;
+
```

Comment:
I don't understand... you mean put ViewableData into a namespace of the same name, so `SilverStripe\ViewableData\ViewableData`?

This pull request only covers Security namespace for the time being.